woocommerce/woocommerce-android
# Display coupon description in coupon details screen

Closes: #6447 

### Description
This PR simply shows the description if it's present in the coupon details.

I don't know if this worth releasing as a hotfix, or if it can wait for 9.2?

### Testing instructions
1. Add description to one of the coupons in wp-admin.
2. Open the coupon details in the app.
3. Confirm the coupon description is shown.

### Images/gif
<img width=360 src="https://user-images.githubusercontent.com/1657201/167005394-a99a7957-b50a-4624-a15a-485690567f1d.png"/>
